seed/static/seed/js/controllers/data_upload_modal_controller.js

```diff
@@ -388,6 +390,7 @@ angular.module('BE.seed.controller.data_upload_modal', [])
         cycle_id = cycle_id || $scope.selectedCycle.id;
         $scope.uploader.in_progress = true;
         save_raw_assessed_data(file_id, cycle_id, true);
+        $rootScope.$emit('datasets_updated')
```

Comment:
trigger `refresh_data()` in the /dataset_list_controller.js after user confirms meters

Reply:
nice!  could we add a semicolon?